WcaOnRails/spec/factories/users.rb

```diff
@@ -16,15 +16,24 @@
     factory :admin do
       name "Mr. Admin"
       email "admin@worldcubeassociation.org"
-      software_team true
+      after(:create) do |user|
+        software_team = FactoryGirl.create(:team, friendly_id: 'software', name: 'Software Team', description: 'Does software')
```

Comment:
I guess none of our tests currently do, but in theory we should have a test like "admins cannot demote themselves, but other admins can". That would require having two admins in a single test.